### DISCUSSION TITLE
Difference between the new

### DISCUSSION THREAD

**Discussion by GridexX** [zricethezav/gitleaks#931] [Q&A] [answered]
Regarding the new 15.2 release of GitLab, Gitleaks was improved to be faster, as decribed [here](https://about.gitlab.com/releases/2022/07/22/gitlab-15-2-released/#faster-secret-detection).

I have search on the repository and the file [`gitleaks.toml`](https://gitlab.com/gitlab-org/security-products/analyzers/secrets/-/blob/master/gitleaks.toml) was updated with new keywords rules.

Comparing to the file on [GitHub](https://github.com/zricethezav/gitleaks/blob/master/config/gitleaks.toml), there are still difference between both.

So I was asking myself what are the differences into the rules between the two files and why ?

**Comment by zricethezav** [comment, marked as answer]
Gitleaks is a dependency of Gitlab's Secret Detection product. Gitlab has different constraints for how rules are managed.
